packages/core/src/js/feedback/utils.ts

```diff
@@ -1,3 +1,18 @@
+import { isFabricEnabled } from '../utils/environment';
+import { ReactNativeLibraries } from './../utils/rnlibraries';
+
+/**
+ * Modal is not supported in React Native < 0.71 with Fabric renderer.
+ * ref: https://github.com/facebook/react-native/issues/33652
+ */
+export function isModalSupported(): boolean {
+  const rnVersion = ReactNativeLibraries.ReactNativeVersion?.version;
```

Comment:
i: We could get the versions on separated variables here.
```suggestion
  const { major, minor } = ReactNativeLibraries.ReactNativeVersion?.version || {};
```

Reply:
Thank you for the suggestion Lucas 🙇